OutputToPwmFactor: 1000.0
ControlLoopFrequency: 500.0
PublishLoopFrequency: 10.0
StartingTiltSafetyLimitDegrees: 10.0
MaxTiltSafetyLimitDegrees: 20.0
ControllerEffortMax: 255
MotorEffortToTorqueSimFactor: 1.0
WheelVelocityAdjustment: 1.0
MadgwickFilterGain: 0.015
MeasuredTiltFilterFrequency: 100.0
MeasuredTiltDotFilterFrequency: 100.0
MeasuredHeadingFilterFrequency: 100.0
MeasuredTurnRateFilterFrequency: 50.0
LeftWheelVelocityFilterFrequency: 10.0
RightWheelVelocityFilterFrequency: 10.0
DesiredForwardVelocityFilterFrequency: 10.0
DesiredTurnRateFilterFrequency: 50.0
MaxVelocityCmd: 4.25 # 3 mph
MaxTurnRateCmd: 6.25 # 150 deg/s
WheelRadiusMeters: 0.05
VelocityCmdScale: 1.15
TurnCmdScale: 6.25
VelocityControlKp: 0.125
VelocityControlKd: 0.05
VelocityControlDerivFilter: 10.0
VelocityControlKi: 0.01
VelocityControlAlphaFilter: 0.99
VelocityControlMaxIntegralOutput: 0.025
VelocityControlOutputLimitDegrees: 8.5
TiltControlKp: 34
TiltControlKd: 0.3
TiltControlAlphaFilter: 0.05
TiltOffset: 0.5
TiltDotOffset: -3.1
RollDotOffset: 0.0
YawDotOffset: 0.0
TurningControlKp: 0.25
TurningControlKi: 0.05
TurningControlKd: 0.0
TurningControlDerivFilter: 50.0
TurningOutputFilter: 0.5